packages/devtools_app/lib/src/screens/debugger/controls.dart

```diff
@@ -154,14 +154,22 @@ class CodeCoverageToggle extends StatelessWidget {
       child: ValueListenableBuilder<bool>(
         valueListenable: controller.codeViewController.showCodeCoverage,
         builder: (context, selected, _) {
+          final isInSmallMode = MediaQuery.of(context).size.width <
+              DebuggingControls.minWidthBeforeScaling;
           return Row(
             children: [
               ToggleButton(
-                label: const Text('Show Coverage'),
+                label: isInSmallMode ? null : 'Show Coverage',
                 message: 'Show code coverage',
                 icon: Codicons.checklist,
                 isSelected: selected,
-                hideBorder: true,
+                outlined: false,
+                shape: const RoundedRectangleBorder(
```

Comment:
why do we have to manually add this for this button and not for all the other buttons in DevTools?